build/lib/ios.js

```diff
@@ -88,7 +88,13 @@ class IOS {
 	async copyLegacyHeaders(DEST_IOS) {
 		// Gather all the *.h files in TitaniumKit, create "redirecting" headers in iphone/include that point to the TitaniumKit ones
 		await fs.ensureDir(path.join(DEST_IOS, 'include'));
-		const subdirs = await fs.readdir(path.join(IOS_ROOT, 'TitaniumKit/build/TitaniumKit.xcframework/ios-arm64_armv7/TitaniumKit.framework/Headers'));
+		let headersDir = path.join(IOS_ROOT, 'TitaniumKit/build/TitaniumKit.xcframework/ios-arm64_armv7/TitaniumKit.framework/Headers');
```

Comment:
This change is for building The SDK itself with Xcode 11 - we didn't copy the headers properly because the directory name didn't match. Now  we do and we want the dev that it'll be missing arm64 sim support.